Assignment-5/app/src/components/ReapeatingDataFetching.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+
+function FetchedData() {
+  const [data, setData] = useState([]);
```

Comment:
This task is not working properly if i click on refresh button it updates the data 2 two times on a single click

Reply:
fixing it